packages/de-code-icons/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "de-code-icons",
+  "version": "1.72.0",
+  "description": "De-code icons",
```

Comment:
de-code-icons is an Open Source CSS icon library, build for modern DevOps SaaS platforms.